src/test/java/com/aws/greengrass/lifecyclemanager/LifecycleTest.java

```diff
@@ -180,12 +180,38 @@ void GIVEN_state_new_WHEN_requestStart_called_THEN_install_invoked() throws Inte
         lifecycle.initLifecycleThread();
         lifecycle.requestStart();
 
-        verify(greengrassService, timeout(1000)).install();
+        verify(greengrassService, timeout(1000).atLeastOnce()).install();
         verify(greengrassService, timeout(1000)).startup();
         assertEquals(State.STARTING, lifecycle.getState());
         assertThat(lifecycle.getStatusDetails(), is(STATUS_DETAIL_HEALTHY));
     }
 
+    @Test
+    void GIVEN_state_new_WHEN_requestStop_called_THEN_shutdown_normally() throws InterruptedException {
+        lifecycle = new Lifecycle(greengrassService, logger, greengrassService.getPrivateConfig());
+        initLifecycleState(lifecycle, State.NEW);
+
+        CountDownLatch installInterrupted = new CountDownLatch(1);
+        Mockito.doAnswer((mock) -> {
+            try {
+                Thread.sleep(1000);
+            } catch (InterruptedException e) {
+                installInterrupted.countDown();
+            }
+            return null;
+        }).when(greengrassService).install();
+
+        lifecycle.initLifecycleThread();
+        lifecycle.requestStart();
+
+        verify(greengrassService,timeout(1000).atLeastOnce()).install();
+
+        lifecycle.requestStop();
+
+        assertThat(installInterrupted.await(1000, TimeUnit.MILLISECONDS), is(true));
+        verify(greengrassService,timeout(1000)).shutdown();
```

Comment:
I don't want shutdown to be called when we're killing `install`. Shutdown is for stopping `startup`/`run` only.